app/routes.py

```diff
@@ -0,0 +1,381 @@
+from app import db
+from flask import Blueprint, jsonify, request
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from datetime import datetime, timedelta
+import os
+
+
+videos_bp = Blueprint("videos_bp", __name__, url_prefix="/videos")
+customers_bp = Blueprint("customers_bp", __name__, url_prefix="/customers")
+rentals_bp = Blueprint("rentals_bp", __name__, url_prefix="/rentals")
+
+# VIDEO ROUTES
+@videos_bp.route("", methods=["POST", "GET"])
+def handle_videos():
+    # POST REQUEST
+    if request.method == "POST":
+        video_request_body = request.get_json()
+        
+        if "title" not in video_request_body:
+            return jsonify({"details": "Request body must include title."}), 400
+        elif "release_date" not in video_request_body:
+            return jsonify({"details": "Request body must include release_date."}), 400
+        elif "total_inventory" not in video_request_body:
+            return jsonify({"details": "Request body must include total_inventory."}), 400
```

Comment:
you could create a function to handle these guard clauses dynamically and then use that function for customers and videos it could look something like
```
def validate_parameters(request_body, parameters):
           for attribute in parameters:
                if attribute not in request_body:
                       abort(make_response({"details": f"Request body must include {attribute}."}, 400))
```